MH4U Dex/ItemAreaSourcesTableViewController.m

```diff
@@ -79,10 +79,12 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
 
 - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath *)indexPath
 {
-    // ToDo: chage to specific area/region once area/region views are in.
     UIStoryboard *storyBoard = [UIStoryboard storyboardWithName:@"Main" bundle:nil];
-    RegionEncyclopediaCollectionViewController *regionVC = [storyBoard instantiateViewControllerWithIdentifier:@"RegionEncyclopediaCollectionViewController"];
-    regionVC.managedObjectContext = self.managedObjectContext;
+    AreaDrop *drop = self.sources[indexPath.row];
```

Comment:
With table view controllers like this, where you're accessing a property array, I often find it useful to declare helper method(s) to get the model object for a given index path—e.g., `- (AreaDrop *)dropAtIndexPath:(NSIndexPath *)indexPath`—to centralize that logic and to give you a correctly-typed result from the method call (rather than by assigning the item into a typed variable).